examples/Cpp/GoogleTest/lib/googletest/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include(FetchContent)
+
+FetchContent_Declare(
```

Comment:
Currently, GoogleTest is downloaded in the CI job:
```yaml
      - name: ⏬ Checkout 'google/googletest' repository
        uses: actions/checkout@v4
        with:
          repository: google/googletest
          path: examples/Cpp/GoogleTest/lib/googletest
```

Looks like a CMake fetch task might be the better solution (@skoehler).
Please also adjust `Pipeline.yml`.

------
How to run CMake to setup the environment (fetching contents) without building the executable?

Reply:
Let's say we are in the build directory. by running `cmake <path_to_the_top_level_cmake_file>` the project is prepared for compiling including fetchcontents being populated.

Reply:
We don't even need to be in the build directory if give the `-B <path_to_build_directory>` to cmake command.